gamemodes/terrortown/gamemode/roles/arsonist/cl_arsonist.lua

```diff
@@ -60,6 +60,7 @@ end)
 -- Show "DOUSED" label on players who have been doused
 hook.Add("TTTTargetIDPlayerText", "Arsonist_TTTTargetIDPlayerText", function(ent, cli, text, col, secondaryText)
     if GetRoundState() < ROUND_ACTIVE then return end
+    if not IsPlayer(ent) then return end
```

Comment:
I think we need to check if this is a corpse too
```suggestion
    if not IsPlayer(ent) and not IsRagdoll(ent) then return end
```